src/sentry/notifications/utils/tasks.py

```diff
@@ -85,6 +104,16 @@ def _send_notification(notification_class_name: str, arg_list: Iterable[Mapping[
                 output_kwargs[arg["key"]] = instance
             else:
                 output_args.append(instance)
+        elif arg["type"] == "lazyobjectuser":
+            if arg["class"] == "RpcUser":
+                user = RpcUser.parse_obj(arg["data"])
+            else:
+                user = User.objects.get(pk=arg["data"]["pk"])
```

Comment:
🤔 This shouldn't be possible as this task is scoped to REGION mode, and User orm is only available in control.